app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -104,7 +104,7 @@
           m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
           })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-          ga('create', 'UA-26179049-6', 'alphagov.co.uk');
+          ga('create', 'UA-26179049-6', '<%= ENV.fetch('GOVUK_APP_DOMAIN') %>');
```

Comment:
Using `fetch` like this means that apps will blow up in development unless they're run using `govuk_setenv`.  Would be good to avoid that if possible.